content/zh-cn/docs/reference/glossary/kubelet.md

```diff
@@ -2,7 +2,7 @@
 title: Kubelet
 id: kubelet
 date: 2018-04-12
-full_link: /zh-cn/docs/reference/generated/kubelet
+full_link: /docs/reference/generated/kubelet
```

Comment:
The en link is redirected to `/docs/reference/command-line-tools-reference/kubelet/`.

Whether the zh-cn link should follow? We may need to modify this file:
https://github.com/kubernetes/website/blob/main/static/_redirects